src/app/shared/entry.ts

```diff
@@ -202,10 +203,26 @@ export abstract class Entry implements OnInit, OnDestroy {
       if (foundVersion) {
         return foundVersion;
       }
+    } else {
+      return null;
     }
 
     // Select newest last_modified version, if it's the same, choose the top
-    return versions.reduce((a, b) => b.last_modified > a.last_modified ? b : a);
+    //  return versions.reduce((a, b) => b.last_modified > a.last_modified ? b : a);
```

Comment:
Think this comes up later, but commented code should not be checked in unless there is a pretty clear `//TODO` or `//FIXME` that indicates in what condition it should be uncommented